lib/ecto/enum.ex

```diff
@@ -279,6 +279,55 @@ defmodule Ecto.Enum do
     |> Keyword.values()
   end
 
+  @doc """
+  Helper to cast values through `cast/2`.
+
+  Provides an ergonomic API to `cast/2`.
```

Comment:
I think `cast/2` is hidden from docs:

```suggestion
  Casts a value from the given `schema` and `field`.
```